source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/APIControllers/UsersTest.cs

```diff
@@ -26,6 +26,24 @@ private async Task<User> GetCurrentUser()
             return user;
         }
 
+        [Fact]
+        public async Task Get_CurrentUser_Includes_IncludedResources()
+        {
+            NeedsConfiguredCurrentUser();
+
+            var response = await Get("/api/users/current-user?include=organization-memberships");
```

Comment:
the include parameter on the query-side can now be dynamic, and we don't need to manually include the world on the backend.